src/components/SocialMedia/social.spec.js

```diff
@@ -0,0 +1,125 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import { BrowserRouter as Router, Route } from 'react-router-dom';
+import { userToken } from '../../../__mocks__/mockData';
+import SocialMediaButtons from './SocialMediaButtons';
+import SocialMediaLogin from './SocialMediaLogin';
+
+const content = {
+  authReducer: {
+    isAuthenticated: true
+  }
+};
+const store = {
+  getState: () => {
+    return { ...content };
+  },
+  subscribe: () => {
+    return {};
+  },
+  dispatch: () => {
+    return {};
+  }
+};
+
+describe('SocialMediaButtons', () => {
+  let wrapper;
+  beforeEach(() => {
+    wrapper = mount(<SocialMediaButtons />);
+  });
+  it('should contain a div with the class social', () => {
+    expect(wrapper.find('div').hasClass('social'));
+  });
+  it('should contain 3 anchor links', () => {
+    const anchorTag = wrapper.find('.social').props().children;
+    expect(wrapper.find('a')).toBeTruthy();
+    expect(anchorTag.length).toEqual(3);
```

Comment:
assert that it renders 3 anchor tags as children.

Currently, you've asserted that it renders 3 children. You've also asserted that there's at least one anchor tag rendered in the component.